test/fixtures/basic/unmanaged-pod.yml.erb

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: unmanaged-pod-<%= deployment_id %>
+  labels:
+    type: unmanaged-pod
+    name: unmanaged-pod-<%= deployment_id %>
+    app: basic
+spec:
+  activeDeadlineSeconds: 20
+  restartPolicy: Never
+  containers:
+    - name: basic
+      image: hello-world:latest
+      imagePullPolicy: IfNotPresent
```

Comment:
We usually use "Always" in production, but in this case we don't really care which version of the image we get, so I'm thinking this should be safe and save test time.